projects/plugins/crm/sass/_ZeroBSCRM.prevadmincore.scss

```diff
@@ -97,7 +97,7 @@ pre {
 }
 
 code,kbd,pre,samp {
-    font-family:monospace,monospace;
```

Comment:
We may want to keep this.

In some browsers, `font-family: monospace` renders smaller, apparently because those browsers have separate preferences for "default proportional font size" and "default monospace font size", and so if the font-size winds up being based on the browser default (including the `1em` here) then it applies the corresponding preference to calculate it. Apparently doing `font-family: monospace, monospace` tricks at least some such browsers to use the proportional preference instead of the monospace preference.

When I tried it out, Firefox 137 and Chromium 135 both show different rendering for `font-family: monospace` versus `font-family: monospace, monospace`.

Reply:
Restored and disabled linting for that one instance: a7e2158c25